src/utility/index.jsx

```diff
@@ -180,6 +180,9 @@ const getTitle = (row, col, heroName) => {
 };
 
 export const getHeroImageUrl = (heroId, imageSizeSuffix) => {
+  if (heroId === 121) {
+    return '/assets/images/grimstroke_sb.png';
```

Comment:
is this necessary? doesn't the dotaconstants update include this path?

Reply:
returns 404 right now. this path points to the image I uploaded manually.

Reply:
oh sorry. i mean, the other image is the big version. it's used for the heroes page for example.